### ISSUE TITLE
Lightweight dist files for modern browsers 

### ISSUE THREAD

**Issue by leplatrem** [Kinto/kinto.js#249]
Currently, *browsersify* builds the dist files with the ES6 polyfill embedded.

The resulting files are quite big. 

```
768K kinto-1.0.0-dev.js
154K kinto-1.0.0-dev.min.js
```

When excluding `babel/polyfill`, it becomes reasonable:

```
371K kinto-1.0.0-dev.js
62K  kinto-1.0.0-dev.min.js
```

We could either:
* build dist files for ES5 appart (`kinto-1.0.0-es5.js`)
* always omit the polyfill from the dist and suggest devs to include it in a separate `<script>` ([as documented in babel](https://babeljs.io/docs/usage/polyfill/))

> Note: `isomorphic-fetch` also weights 30K unminified :)

Blocks: #272

**Comment by n1k0** [comment]
Great idea. I'm really uncomfortable with the filesize of the generated assets anyway.

Once done, we should also document how to use common polyfills provided by third-party providers, eg. http://polyfills.io/.

**Comment by leplatrem** [comment]
Fixed in #291, v1.1.2
